src/test/java/picard/fingerprint/CrosscheckReadGroupFingerprintsTest.java

```diff
@@ -0,0 +1,300 @@
+package picard.fingerprint;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.vcf.SamTestUtils;
+
+import java.io.*;
+import java.lang.reflect.Field;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static picard.fingerprint.FingerprintIdDetails.multipleValuesString;
+
+/**
+ * Tests for CrosscheckReadGroupFingerprints
+ */
+public class CrosscheckReadGroupFingerprintsTest {
```

Comment:
Would be nice to avoid code duplication with `CrosscheckFingerprintsTest` if possible

Reply:
I kept it as is on purpose...the idea is that CrosscheckReadgroupFingerprints is deprecated, and so I revived the original CrosscheckReadgroupFingerprints tests as proof that the refactoring didn't change anything....once the deprecation becomes a removal, there will be no code-duplication...does that make sense?

Reply:
OK